### PR TITLE
refactor(animations): improve unit tests for stagger timing

### PR DESCRIPTION
**Pull request by dario-piotrowicz**
improve the stagger timing unit tests added in #47208 by also checking
that the duration and keyframes of the players are correct

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information
 
 - Just a small unit tests improvement as suggested by @jessicajaniuk in #47208 :slightly_smiling_face:

### CODE REVIEW

File: packages/core/test/animation/animation_query_integration_spec.ts [hunk 1]
```diff
@@ -783,17 +786,23 @@ describe('animation query tests', function() {
...
-    it(`should handle params used in the stagger's timing argument producing a negative value`,
-       () => {
-         @Component({
+    it(
+        `should handle params used in the stagger's timing argument producing a negative value`,
+        () => {
+          const animationDuration = 700;
+          const absoluteStaggerDelay = 500;
+
+          @Component({
```
Comment (dario-piotrowicz) [inline packages/core/test/animation/animation_query_integration_spec.ts:805]: this gets wrongly indented by the formatter :sweat:

Comment (dario-piotrowicz) [reply packages/core/test/animation/animation_query_integration_spec.ts:805]: ok I managed to fix it by shortening the `it` text :slightly_smiling_face:

**Review by jessicajaniuk — APPROVED** [review]
LGTM :cookie:

Thanks, @dario-piotrowicz!

**Comment by dario-piotrowicz** [comment]
@jessicajaniuk I'm closing the PR as I will have to create a new one to fix the stagger issue you've found, so I guess it just makes more sense for me to include the updated unit tests in my next PR :slightly_smiling_face:

**Pull request closed without merging**